.travis.yml

```diff
@@ -41,7 +37,7 @@ before_install:
       echo -e "nhibernate = /tmp/firebird/nhibernate.fdb" | sudo tee -a /opt/firebird/databases.conf
       echo -e "AuthServer = Srp\nAuthClient = Srp\nUserManager = Srp\nWireCrypt = Enabled" | sudo tee -a /opt/firebird/firebird.conf
       sudo /opt/firebird/bin/gsec -modify SYSDBA -pw masterkey -admin yes
-      sudo service firebird restart
+      sudo systemctl restart firebird-superserver
```

Comment:
The reason it was failing is because the service name on 3.0.x has changed. Probably 14.04 has some residue service definitions for `firebird` service and so it was working as intended.